spec/factories/passengers.rb

```diff
@@ -18,9 +18,9 @@
       doctors_note nil
     end
 
-    trait :recently_expired do
+    trait :expired_within_grace_period do
```

Comment:
The seed file *does* need to be changed. It now references a factory which no longer exists.

Reply:
Ooops